.github/README.md

```diff
@@ -9,7 +9,7 @@ Oppia is written using Python and AngularJS, and is built on top of Google App E
   * [Oppia.org community site](https://www.oppia.org)
   * [User Documentation](https://oppia.github.io/)
   * [Contributors' wiki](https://github.com/oppia/oppia/wiki)
-  * [Developer mailing list](http://groups.google.com/group/oppia-dev)
```

Comment:
Why are we removing Developer mailing list?

Reply:
@gp201 Because I don't think we want this to be the main contact point. It seems better to point folks to GitHub Discussions as the point of contact to raise things, right?

Reply:
@seanlip @gp201 I have to change something?

Reply:
I don't think so, but @gp201 should confirm.
@gp201 what are your thoughts on my comment above?

Reply:
@seanlip all the necessary developer announcements come in this channel. So we need to keep this mailing list. Maybe we call it developer announcements instead?

Reply:
Yup ok, that works, thanks!

@MohitGupta121 could you please change "mailing list" to "announcements"?

Reply:
Okay fine @seanlip from "Developer mailing list" to "Announcements" Or "Developer announcements"?

Reply:
Developer Announcements

Reply:
Okay thanks. I'll update it.

Reply:
Done @seanlip @gp201